index.html

```diff
@@ -196,5 +196,39 @@ <h3>Skills</h3>
         </section>
         
        
+        <section class="contact-me">
+          <div class="container get-in-touch">
+            <p class="para">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+          </div>
+        </section>
+
+      
+          <form class="form" action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] I will encourage you to stick to the design, use the fonts, layout and flexbox so that your app would look like the Figma's.

### Figma project
![image](https://github.com/Nessrine88/-Protofolio-/assets/86200356/f70b5917-9d95-4efa-981a-163cce19c101)

### Your project
![image](https://github.com/Nessrine88/-Protofolio-/assets/86200356/56c475b7-1518-42ad-ab34-5c95efc0874f)